backend/codesirius/problems/views/tag.py

```
@@ -0,0 +1,82 @@
+            raise NotFound("Tag not found")
+        except Exception as e:
+            logger.error(f"Error while fetching tag: {e}")
+            raise APIException(
+                detail="Error while fetching tag",
+                code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            )
+
+        serializer = TagSerializer(tag)
+        return CodesiriusAPIResponse(data=serializer.data)
+
+    def put(self, request, pk):
```

Ruff (0.8.2)

Missing return type annotation for public function `put`

(ANN201)